test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -306,6 +306,11 @@ public static double randomDoubleBetween(double start, double end, boolean lower
         return result;
     }
 
+    /** A random float between 0.0 and 1.0 (inclusive). */
+    public static float randomFloatRatio() {
```

Comment:
@cbuescher The issue with randomFloat() is that it could return any float number, positive or negative and in any range.  I wanted something that gives a float between 0.0 and 1.0, and I figured many of our API values are ranges between 0.0 and 1.0.  Do you think it warrants its own call  in ESTestCase or should I just put the same functionality in SuggestSearchTests?